src/components/common/Conversation.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { object } from 'prop-types';
+
+import profilePic from '../../img/guy.svg';
+
+const Conversation = ({
+  conversation: { unreadMessages, user, lastMessage, topicIcon }
+}) => (
+  <div className="row conversation">
+    {unreadMessages != 0 &&
```

Comment:
You can do `unreadMessages &&`

Reply:
It's a string, so when "0" evaluates to true.